features/addressing/managing_zones/deleting_zone.feature

```diff
@@ -0,0 +1,24 @@
+@managing_zones
+Feature: Deleting a zone
+    In order to remove test, obsolete or incorrect zones
+    As an Administrator
+    I want to be able to delete a zone
+
+    Background:
+        Given the store has a zone "North America" with code "NA"
+        And the store also has a zone "South America" with code "SA"
+        And the store has a tax category "Sports gear"
+        And the store has "European Union Sales Tax" tax rate of 20% for "Sports gear" within "SA" zone
```

Comment:
EU Sales Tax in South America? That would be interesting. Our examples should be at least close to reality. :D